### DISCUSSION TITLE
When will a v4 compatible Figma UI Kit be available?

### DISCUSSION THREAD

**Discussion by minkyn** [primefaces/community#2024] [Figma UI Kit]
PrimeVue v4's token set seems quite different from available in v3, missing many variables such as "--text-color", "--surface-ground", "--font-family", etc. Without an available v4 compatible Figma UI Kit, it is really difficult for teams to start transitioning to v4.

**Comment by onursenture** [comment]
V4 is still in beta, but we are continuing our work on V4 compatibility in Figma. Considering that the implementation of theming is also ongoing, it would not be accurate to provide an estimate now.
